vscode.opam

```diff
@@ -10,10 +10,11 @@ doc: "https://tmattio.github.io/js-bindings/"
 bug-reports: "https://github.com/tmattio/js-bindings/issues"
 depends: [
   "ocaml" {>= "4.08.0"}
-  "dune"
-  "ojs"
   "es5" {= version}
+  "dune"
+  "gen_js_api"
```

Comment:
I purposefully avoided the dependency to `gen_js_api` in the opam packages. It's used only during development to generate the implementations, and the files are then promoted. We can install it manually in the CI

Reply:
Indeed, I just changed that.